The great datka, KvaGram, demands attention!

KvaGram opened the pull request:
Quack!

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

Commit:
I am KvaGram the great datka

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,2 @@
+I am KvaGram, the great datka.
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

KvaGram replied:
A list of the recently slain has been added!

Commit:
Added a list of the recently(ish) slain!

The great datka has slain many code-enemies!
The big beast GitHub is next.

Commit:
The gofers of golang have been added to the list

Commit:
My deadly skills

KvaGram committed:
Merge pull request #1 from KvaGram/deadly_skills

My deadly skills

Commit:
just so there is another pull request

Commit:
adding dark arts

Really don't think letting this be known is a good idea.

codeschool-kiddo committed:
Added new rule about coffee

Commit:
Fixed spelling

I must say I disapprove of this new rule. Coffee is life. Coffee is
king!
And then you misspell it? Is insulting me a part of the test, master?

KvaGram committed:
Merge pull request #3 from KvaGram/new_rules

Too many people are spilling coffee

Commit:
Merge branch 'deadly_skills'

Commit:
Kill bad code, not people

Commit:
Merge tag 'v1.0.1'

codemurder
